README.md

```diff
@@ -1,34 +1,25 @@
-# node-controller-manager
+# Node-Controller-Manager
 
-The Node Controller Manager bundles multiple Kubernetes controllers, that help with the creation, maintance and operation of different instance (node) Kubernetes resources. 
+Node Controller Manager (NCM) manages VMs as another kubernetes custom resource. It provides a declarative way to manage VMs. The current implementation only supports AWS but can be (and will be) easily extended to support other cloud providers.
 
-It's designed to run in master plane of a kubernetes cluster and follows the best principles and practices of writing controllers, including, but not limited to:
-
-- leader election to allow HA deployments of the controller
-- `workqueues` and multiple thread-workers
-- `SharedInformers` that limit to minimum network calls, deserialization and provide helpful create/update/delete events for resources
-- rate-limiting to allow back-off in case of network outages and general instability of other cluster components
-- sending events to respected resources for easy debugging and overview
-- Prometheus metrics, health and (optional) profiling endpoints
-
-
-
-## Build
-
-```bash 
-go build -i cmd/node-controller-manager/controller_manager.go
+Example of managing VM:
 ```
+kubectl create/get/delete instance vm1
+``` 
 
-## Running
+### Key terminologies
 
-For testing/development purposes, you can create your own kubernetes cluster and download its kubeconfig file. Use this kubeconfig file to be passed as an argument to your node-controller-manager. You can run the node-controller-manager on your local machine as mentioned below
+Nodes/Instances/Machines/VMs are different terminologies used to represent the similar things in theory. However, for most of our discussion the following are our interpretations.
```

Comment:
s/the similar things in theory/similar things/

s/However, for most of our discussion the following are our interpretations./We use these terms in the following way/

Reply:
Done.
https://github.com/gardener/node-controller-manager/commit/95c68a4c3f483a5f770c7de61f12b25e5411f3c0#diff-04c6e90faac2675aa89e2176d2eec7d8R12